.changeset/sweet-penguins-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`IconTile` - Fixed a11y issues present in the visual design
```

Comment:
Nit: we could make the changes more explicit for consumers (this suggestion would also apply to the doc site version history)

```suggestion
`IconTile` - Updated visual design of IconTile to make it distinguishable from secondary IconButton.
```